src/components/Camera/index.js

```diff
@@ -31,7 +31,7 @@ const CameraPure = ({
   renderTitle, renderError, children,
   webcamRef, onUserMedia, onFailure, video,
 }: Props) => (
-  <div className={classNames(style.camera, className)}>
+  <div className={classNames(style.camera, className)} tabIndex="0" aria-label="View from selfie camera">
```

Comment:
We cannot have hardcoded strings otherwise if the client uses a different language this string will not be updated.

Reply:
Why do you need `tabIndex="0"`?

Reply:
Oops. Will update the code to use the language strings map instead.

For the `tabIndex="0"` that is to get the screenreader to pick up the face preview area as part of the screen's tab order. Currently it isn't being picked up at all in the tab order.

Reply:
I don't think that the camera view should be a tabbable element because it's not an element the user can interact with. Only focusable elements should be reachable via tab, otherwise it might generate confusion when the user tries to interact with it using tab + enter. I think it should only have `aria-label` and it can still be reached using the screen reader navigation (arrow commands).

Reply:
Ah, I see what you mean. Will remove it.